Misc/python.man

```diff
@@ -203,13 +204,13 @@ __debug__; augment the filename for compiled (bytecode) files by
 adding .opt-1 before the .pyc extension.
 .TP
 .B \-OO
-Do \fB-O\fP and also discard docstrings; change the filename for
+Do \fB\-O\fP and also discard docstrings; change the filename for
 compiled (bytecode) files by adding .opt-2 before the .pyc extension.
 .TP
 .B \-P
 Don't automatically prepend a potentially unsafe path to \fBsys.path\fP such
-as the current directory, the script's directory or an empty string. See also the
-\fBPYTHONSAFEPATH\fP environment variable.
+as the current directory, the script's directory or an empty string.
+See also the \fBPYTHONSAFEPATH\fP environment variable.
```

Comment:
This is just reformatting, please revert.

Reply:
Reverted those bits.